ee/app/livechat-enterprise/client/views/app/customTemplates/visitorEditCustomFields.js

```diff
@@ -0,0 +1,28 @@
+import { ReactiveVar } from 'meteor/reactive-var';
+import { Template } from 'meteor/templating';
+
+import { APIClient } from '../../../../../../../app/utils/client';
+import './visitorEditCustomFields.html';
+
+Template.visitorEditCustomFields.helpers({
+	availablePriorities() {
+		return Template.instance().priorities.get();
+	},
+	roomPriority() {
+		return Template.instance().roomPriority.get();
+	},
+});
+
+Template.visitorEditCustomFields.onCreated(async function() {
+	this.priorities = new ReactiveVar([]);
+	this.roomPriority = new ReactiveVar({});
+	const { omnichannel } = this.data;
+
+	if (omnichannel && omnichannel.priority) {
+		this.roomPriority.set(omnichannel.priority);
+	}
+	this.autorun(async () => {
```

Comment:
Does this need to room inside an `autorun` method?
I can't see any `ReactiveVar` that would change.